services/_shared/llm_client.py

```
@@ -367,6 +367,19 @@ def _load_openrouter_key() -> str:
 
 def _review_backend_config(backend: Backend) -> BackendConfig:
     """Return backend settings scoped to the expensive Elder review pass."""
+    if backend == Backend.CAVE:
+        # Cave-primary: the owned in-cluster spark-gateway is the sole review
+        # backend. Raise (not KeyError) when unconfigured so review_diff's
+        # _BackendConfigError arm records it as a clean backend failure.
+        cave = _cave_review_config()
+        if cave is None:
+            raise _BackendConfigError("GRUG_CAVE_GATEWAY_URL not set - cannot run Cave review")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)